### PR TITLE
New package: PRQL.prqlc version 0.9.0

### PR DESCRIPTION
**Pull request by eitsupi**
###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/113133&drop=dogfoodAlpha

### CODE REVIEW

**Comment by stephengillie** [comment]
Hi @eitsupi,

This package has a dependency:
image

File: manifests/p/PRQL/prqlc/0.9.0/PRQL.prqlc.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+PackageIdentifier: PRQL.prqlc
+PackageVersion: 0.9.0
+InstallerLocale: en-US
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: prqlc.exe
+UpgradeBehavior: install
+ReleaseDate: 2023-07-25
+Installers:
```
Comment (stephengillie) [inline manifests/p/PRQL/prqlc/0.9.0/PRQL.prqlc.installer.yaml:13]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

**Comment by eitsupi** [comment]
@stephengillie Thanks for pointing out that!
Ideally, the dependency should not exist, so let's see if we can resolve this.

**Comment by stephengillie** [comment]
The application is written to use the binaries from the dependency. I believe bundling the binaries inside the installer might allow for a dependency-free install.

**Comment by eitsupi** [comment]
It seems that this error is common of Rust packages.
I tried to update the build workflow on PRQL/prql#3086 and it seems that fix the error.
(I tested the artifact on Windows Sandbox)

When PRQL/prql#3086 is merged, I will close this PR and will open another one to point the new release.

**Comment by eitsupi** [comment]
Closing in favor of #114768

**Pull request closed without merging**